src/js/hyperschedule.js

```diff
@@ -1690,6 +1744,9 @@ function readStateFromLocalStorage()
   gShowClosedCourses = readFromLocalStorage(
     "showClosedCourses", _.isBoolean, true
   );
+  gGreyConflictCourses = readFromLocalStorage(
+    "greyConflictCourses", _.isString, greyConflictCoursesOptions[1]
+  );
```

Comment:
I feel like this should be validated.

Reply:
Sorry what do you mean by this?

Reply:
If there's an invalid value in the local storage, we shouldn't just assign it to `gGreyConflictCourses`. Otherwise it becomes much harder to reason about the code. If the value is invalid we should just reset to the default.

Reply:
I think this is resolved in the newest commit:
`gGreyConflictCourses = readFromLocalStorage(
    "greyConflictCourses", _.isString, greyConflictCoursesOptions[0]
  );`

Reply:
I am pretty sure that with this code, if `greyConflictCourses` has a value of `"asdf"` in local storage, then that value will just get assigned to the global variable.

Reply:
Ahh sorry I didn't quite understand before.
This should be resolved in the newest commit.